src/pybind/mgr/dashboard/frontend/src/app/shared/components/custom-login-banner/custom-login-banner.component.scss

```diff
@@ -0,0 +1,5 @@
+.login-banner {
+  font-weight: bold;
+  margin: 0;
+  padding: 12px 20% 12px 12px;
+}
```

Comment:
This is ok, but in general rather than using per-component CSS, we should aim at using site-wide CSS classes, ideally Bootstrap ones, like the following:
* [`.font-weight-bold`](https://getbootstrap.com/docs/4.6/utilities/text/#font-weight-and-italics)
* [`.text-wrap`](https://getbootstrap.com/docs/4.6/utilities/text/#text-wrapping-and-overflow) and even [`.text-break`](https://getbootstrap.com/docs/4.6/utilities/text/#word-break).
* [`.p-1`, `.p-2`, ... `.p-5`](https://getbootstrap.com/docs/4.6/utilities/spacing/#examples) classes add padding around an element
* `.m-0` removes margin.

Or even better, an existing Dashboard class that already does this?:
* `.bold`
*

Reply:
Before this bootstrap classes (like p-1, font-weight-bold) were used. Had to switch to per-component CSS for the `20% right padding` (which I am not sure is possible using Bootstrap classes) needed so that the login banner text doesn't appear over the octopus logo.